atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -462,7 +462,7 @@ private static void checkInstallConfig(AtlasDbConfig config) {
                 || config.getSweepReadLimit() != null
                 || config.getSweepCandidateBatchHint() != null
                 || config.getSweepDeleteBatchHint() != null) {
-            log.error("Your configuration specifies sweep parameters on the install config. They will be ignored."
+            log.warn("Your configuration specifies sweep parameters on the install config. They will be ignored."
```

Comment:
This message (even at ERROR) didn't save one of our internal deployments from believing their install config worked. 😞 

I think for the most part I think of logs as something to guide diagnosis or investigation, not so much to be looked at in and of themselves...

Reply:
Separate task: this should possibly be a healthcheck warning rather than a warning hidden in the logs. It's not severe enough to force a user to look at it (e.g. by refusing to start), but is still something that should be fixed, because ignoring it could lead people to believe their config is different to what it actually is. This would cause:
- Best case: less performant than intended sweep
- Average case: lost time in investigating an issue because it's not obvious enough what the problem is
- Worst case: full stack failure (in case the configuration options are set in order to avoid a Cassandra OOM or similar)